src/transformers/models/auto/configuration_auto.py

```diff
@@ -749,13 +749,13 @@ def from_pretrained(cls, pretrained_model_name_or_path, **kwargs):
             return config_class.from_pretrained(pretrained_model_name_or_path, **kwargs)
         elif "model_type" in config_dict:
             config_class = CONFIG_MAPPING[config_dict["model_type"]]
-            return config_class.from_dict(config_dict, **kwargs)
+            return config_class.from_dict(config_dict, **unused_kwargs)
```

Comment:
We replicate the logic inside `from_pretrained` here, but duplicating some of the kwargs.

Reply:
Sounds good